started/demo.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: Run Fyne Demo
+
+---
+
+If you want to see the Fyne toolkit in action before you start to code your own application,
+you can see our [demo app](https://github.com/fyne-io/fyne/tree/master/cmd/fyne_demo) application.
+
+### Running
+
+If you want to, you can run the demo directly using the following command (requires Go 1.16 or later):
+
+    $ go run fyne.io/fyne/v2/cmd/fyne_demo@latest
+
+For earlier versions of Go, you need to use the following command instead:
+
+    $ go run fyne.io/fyne/v2/cmd/fyne_demo
+
+By browsing the different tabs of the app you can see all the features of Fyne toolkit.
+
+### Installing
+
+It is possible to install the app as a graphical application like all others on your computer.
+We have the helpful `fyne` tool to do this for you.
+First you will need to install the tool:
+
+	go install fyne.io/fyne/v2/cmd/fyne@latest
```

Comment:
Should we be a bit more consistent with `$` at the beginning of commands?

Reply:
Good catch. And in other pages they have been removed, so the ones where it is not present should be updated.
The codeblock style takes care of it - the note was that "$" gets in the way of copy/paste